src/mozilla_sec_eia/models/sec10k/ex_21/inference.py

```diff
@@ -275,21 +295,31 @@ def extract_filings(
 
         logits = []
         predictions = []
+        # hidden_states = []
```

Comment:
I haven't worked out the logic for using the hidden states (from the last layer of the model) yet, but I kept the draft of this commented out in this PR.